brave/brave-core
# Adds Rewards UI spec tests to unit test job

Fixes: https://github.com/brave/brave-browser/issues/6097

This adds the Rewards UI component spec tests to the `test-unit` npm script. This also includes a one-time update of the spec snapshots, as they now will need to use the core generated classnames

<img width="787" alt="Screen Shot 2019-09-19 at 8 57 06 PM" src="https://user-images.githubusercontent.com/8732757/65298363-13d9fb80-db20-11e9-930c-8c1fc581c522.png">

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Pull down this branch
2. Run `npm run test-unit`
3. Confirm that suite passes, note the passing spec tests.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
